src/displayBackend/wayland/Display.cpp

```diff
@@ -437,9 +437,23 @@ void Display::registryHandler(wl_registry *registry, uint32_t id,
 	LOG(mLog, DEBUG) << "Registry event, itf: " << interface << ", id: " << id
 					 << ", version: " << version;
 
+#ifdef WITH_WAYLAND_PRESENTATION_API
+	if (interface == wp_presentation_interface.name) {
+		mPresentation =
+			(wp_presentation*)wl_registry_bind(registry, id, &wp_presentation_interface, 1);
```

Comment:
Do you need to un-bind this during object destruction?

Reply:
yes, there is a wp_presentation_destroy that shold solve this